cuebot/scheduler-sim/analysis/make_graphs_ba.py

```
@@ -0,0 +1,87 @@
+def orphan_series(tag):
+    rx = re.compile(r"\[(\d\d:\d\d:\d\d)\].*orphan=\s*(\d+)"); pts = []
+    try:
+        for ln in open(f"{CMP}/{tag}_sim.log", errors="ignore"):
+            m = rx.search(ln)
+            if m: pts.append((sod(m.group(1)), int(m.group(2))))
+    except FileNotFoundError: return []
+    if not pts: return []
+    t0 = pts[0][0]; return [(t-t0, v) for t, v in pts if 0 <= t-t0 <= WATCH]
+def rows(tag):
+    out = []
+    try: f = open(f"{CMP}/{tag}_dbstat.csv", errors="ignore")
```

Ruff (0.16.0)

[error] Multiple statements on one line (colon)

(E701)